rental.rb

```diff
@@ -0,0 +1,12 @@
+# Student class inherits from Person class
+class Rental
+  attr_accessor :date
```

Comment:
- [ ] `book` and `person` should be marked with `attr_accessor` in _Rental_ class, as we intend to both _access (get)_ as well as _modify (set)_ it. Kindly ensure to improve upon this aspect.